### PR TITLE
Feature/inv 34

### PR DESCRIPTION
**Pull request by jennshevach**
The API Blueprint file was updated to the proper host URL, and several changes were made to support the addition of manufacturer endpoints. There were also a few minor fixes made throughout. Additionally, the nuspec file was updated to reflect two dependent applications' version numbers.

Manufacturers are a sub-collection of inventory items. As such, the data structure for inventory items was updated to include a count of manufacturers on a GET and GETALL.

For the manufacturers collection, GET and POST endpoints were added. For the detail, GET, POST, and DELETE were added. Data structures to support the manufacturers and their associated part numbers were also added.

### CODE REVIEW

File: DudeInventoryApi.PCL.nuspec [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <version>$version$</version>
     <authors>Dude Solutions, Inc.</authors>
     <owners>Dude Solutions, Inc.</owners>
-    <copyright>Copyright � 2015 Dude Solutions, Inc. - All Rights Reserved</copyright>
+    <copyright>Copyright � 2015 Dude Solutions, Inc. - All Rights Reserved</copyright>
```
Comment (aoffpo) [inline DudeInventoryApi.PCL.nuspec]: Fix copyright © symbol.

File: apiary.apib [hunk 2]
```diff
@@ -122,6 +122,55 @@ This endpoint allows for the select, update, and delete of specific inventory it
 
 + Response 204
 
+# Group Manufacturer
+
+## Manufacturer Collection [/accounts/{accountId}/inventoryitems/{inventoryItemId}/manufacturers]
+This endpoint allows for the retrieval of a list of manufacturers associated with a specific inventory item
```
Comment (aoffpo) [inline apiary.apib:124]: Great job on the descriptions!

File: apiary.apib [hunk 3]
```diff
@@ -175,4 +224,47 @@ Represents the inventory item data with all fields except the id which will be r
 Represents the inventory item with all fields including the system generated id
 
 ###Properties
-+ `id`: 1234567890 (number, required) - {INT} The ID for the object
\ No newline at end of file
++ `id`: 1234567890 (number, required) - {INT} The ID for the object
+
+
+## Inventory Item Manufacturer (Inventory Item Update)
+Represents the inventory item will all fields including the ID for the object and a count of the associated manufacturers.
```
Comment (aoffpo) [inline apiary.apib]: will means with?

**Comment by aoffpo** [comment]
+1

**Comment by aoffpo** [comment]
+1

**Comment by jwholme2-dude** [comment]
+1

**Comment by jwholme2-dude** [comment]
+1

**Pull request merged**